core/src/epicli/cli/epicli.py

```diff
@@ -460,7 +460,6 @@ def dump_external_debug_info(title, args):
         dump_external_debug_info('ANSIBLE CONFIG', ['ansible-config', 'dump'])
         dump_external_debug_info('ANSIBLE-VAULT VERSION', ['ansible-vault', '--version'])
         dump_external_debug_info('TERRAFORM VERSION', ['terraform', '--version'])
-        dump_external_debug_info('SKOPEO VERSION', ['skopeo', '--version'])
```

Comment:
Nope, as I decided not to wrap the the Crane binary inside a custom python package like we did for Skopeo:

https://github.com/epiphany-platform/skopeo-bin

Since Crane never run inside the container itsself but only copied to the repo machine or being returned by ```epicli prepare ...``` in the requirements package.